### PR TITLE
Establish open Governance for the OpenCSM project.

### PR DESCRIPTION
**Pull request by alexlovelltroy**
This pull request removes significant portions of the HPE governance and replaces them with a simplified structure suitable for an open consortium.

### CODE REVIEW

**Comment by eric-roman** [comment]
The largest issue I see with the governance so far is the representation of the user community.
So far the governance addresses the interests of the founding members (via the board) and
the developers (via the TSC) but the success of the effort relates to the software's
to meet user needs.  What's the best way to accomplish this in the governance structure?

**Comment by alexlovelltroy** [comment]
@eric-roman That's an excellent point.  We haven't clearly defined who the users of OpenCSM are.  I have been operating under the impression that our users are the operators of HPC systems.  Each of the founding members has their own layer above OpenCSM that exposes compute resources through tenancy or other means to the users of those systems.  Should we expand our scope to include HPC users that submit jobs as direct OpenCSM users?

File: README.md [hunk 1]
```diff
@@ -16,20 +16,4 @@ For more specific topics, try a SIG.
 ---
 Note that this repository is loosely modeled on the kubernetes open governance repository at https://github.com/kubernetes/community.  The goal is to apply the similar governance within the Cray / HPE community.  Phase 1 will seed the governance and Phase 2 will expand participation.
 
-There will be two phases for the rollout of over-all governance for the Cray / HPE community:
-
-### Phase 1 
-Our initial model will be very lean and simple. Community participants may submit PR's against any one of the published repositories. Once the PR is submitted, the Cray internal engineering management for that component/repo will review the PR and determine its suitability. Accepted PR's will be merged to master and become part of a subsequent release within the context of release cadence as it exists as of November 2020. Decisions regarding community contributions to a given repository will be at the sole discretion of the Cray engineering organization during this phase.
```
Comment (alamsr01) [inline README.md:22]: Cray->HPE

**Comment by alamsr01** [comment]
In my view, the concept of the users of OpenCSM can evolve over time.  It should be somewhat transparent to a classic HPC usage scnearios such as submitting job, login, etc.  We can start with the roles such as system and network engineers, platform developers and operators who are obvious stakeholders and likely acutely aware of (Open)CSM interfaces. The current SIGs can be a starting point for the user communities.

File: README.md (hunk 1 above)
Comment (larry-kaplan) [reply README.md:22]: This appears to be a deletion of old stuff, being replaced by a link to the original prose at its source.  So nothing really to fix here I think though maybe the original source could use the fix!

**Comment by alexlovelltroy** [comment]
I've updated README.md to clarify who our users are.  @alamsr01 and @eric-roman can you comment and/or approve?

**Review by alamsr01 — APPROVED** [review]

**Comment by alexlovelltroy** [comment]
I believe this is ready for merge.  @eric-roman if you are satisfied, can you approve?  We can amend later by board action if needed.

**Comment by eric-roman** [comment]
Sure.  The proposed changes look good and those comments can be addressed in another issue.  The specific language of concern is in the TSC paragraph, where votes on TSC membership votes are given to "project development community" without explicit mention of e.g. the operators using the project but not developing code to it.

**Review by eric-roman — APPROVED** [review]

**Pull request merged**